app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptions.kt

```diff
@@ -90,8 +91,13 @@ class ChooseDirectionOptions constructor(private val route: Route?) : DialogFrag
                     null,
                     EmptyDirectionsGuard(
                         CallbackDirectionsConfirmListener { start, end ->
-                            //Display the directions time
-                            route?.set(start, end)
+                            // Start the DirectionsActivity activity
+                            val intent = Intent(this.context, DirectionsActivity::class.java).apply {
+                                putExtra("Origin", start)
+                                putExtra("Destination", end)
+                            }
+                            startActivity(intent)
+                            this.dismiss()
                         },
                         DisplayMessageErrorListener(this.requireActivity()))
                 )
```

Comment:
I'll take care of that, but I think we'll be able to remove that dialog altogether

Reply:
Sounds good